### PR TITLE
Decompile birch_pc.c

### PR DESCRIPTION
**Pull request by KDSKardabox**
First commit. So a bit short.

### CODE REVIEW

File: src/birch_pc.c [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+extern const u8 gBirchDexRatingText_LessThan120[];
+extern const u8 gBirchDexRatingText_LessThan130[];
+extern const u8 gBirchDexRatingText_LessThan140[];
+extern const u8 gBirchDexRatingText_LessThan150[];
+extern const u8 gBirchDexRatingText_LessThan160[];
+extern const u8 gBirchDexRatingText_LessThan170[];
+extern const u8 gBirchDexRatingText_LessThan180[];
+extern const u8 gBirchDexRatingText_LessThan190[];
+extern const u8 gBirchDexRatingText_LessThan200[];
+extern const u8 gBirchDexRatingText_DexCompleted[];
```
Comment (Diegoisawesome) [inline src/birch_pc.c]: Move these declarations into text.h and include that

File: src/smokescreen.c [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#include "global.h"
+#include "data2.h"
+#include "decompress.h"
+#include "sprite.h"
+#include "util.h"
+
+
+static void sub_8075370(struct Sprite *);
+
```
Comment (Diegoisawesome) [inline src/smokescreen.c]: Remove these extra newlines

File: src/birch_pc.c (hunk 1 above)
Comment (KDSKardabox) [reply src/birch_pc.c]: done

File: src/smokescreen.c (hunk 2 above)
Comment (KDSKardabox) [reply src/smokescreen.c]: Done.

File: src/birch_pc.c [hunk 3]
```diff
@@ -3,33 +3,12 @@
 #include "field_message_box.h"
 #include "pokedex.h"
 #include "constants/species.h"
+#include "text.h"
```
Comment (Diegoisawesome) [inline src/birch_pc.c]: I meant strings.h, sorry!

**Pull request merged**